duckdb-rfuns-r/tests/testthat/test-generated-plus.R

```diff
@@ -0,0 +1,191 @@
+# Generated by helper-gen.R: do not edit by hand
+test_that('<dbl> + <dbl> :: 1 + 1', {
+  con <- local_con()
+  in_df <- tibble::tibble(x1 = 1, x2 = 1)
+  in_rel <- duckdb:::rel_from_df(con, in_df)
+  out_rel <- duckdb:::rel_project(
+    in_rel,
+    list(duckdb:::expr_function(
+      'r_base::+',
+      list(
+        duckdb:::expr_reference('x1'), 
+        duckdb:::expr_reference('x2')
+      )
+    ))
+  )
+  out_df <- duckdb:::rel_to_altrep(out_rel)
+
+  expect_identical(out_df[, 1], 1 + 1)
+})
+
+test_that('<dbl> + <dbl> :: 1e+06 + 2e+06', {
+  con <- local_con()
+  in_df <- tibble::tibble(x1 = 1e+06, x2 = 2e+06)
+  in_rel <- duckdb:::rel_from_df(con, in_df)
+  out_rel <- duckdb:::rel_project(
+    in_rel,
+    list(duckdb:::expr_function(
+      'r_base::+',
+      list(
+        duckdb:::expr_reference('x1'), 
+        duckdb:::expr_reference('x2')
+      )
+    ))
+  )
+  out_df <- duckdb:::rel_to_altrep(out_rel)
+
+  expect_identical(out_df[, 1], 1e+06 + 2e+06)
```

Comment:
Merging now, but in a separate PR perhaps:

```suggestion
  expect_identical(out_df[, 1], 3e+06)
```